common/subdivisions/en.xml

```diff
@@ -5574,7 +5574,7 @@ Generated by tool:	SubdivisionNode
 			<subdivision type="zafs">Free State</subdivision>
 			<subdivision type="zagp">Gauteng</subdivision>
 			<subdivision type="zagt">Gauteng</subdivision>
-			<subdivision type="zakzn">Kwazulu-Natal</subdivision>
+			<subdivision type="zakzn">KwaZulu-Natal</subdivision>
```

Comment:
isn't this part of #913 ?

Reply:
I don't believe so. I have already squashed and merged the PR for Kwazulu-Natal